### PR TITLE
Added FE documentation

### PR DESCRIPTION
**Pull request by giobart**

### CODE REVIEW

**Review by danimair9 — APPROVED** [review]

File: content/docs/dashboard/features.md [hunk 1]
```diff
@@ -16,32 +16,68 @@ The user interface is still under development and therefore does not offer so ma
...
+In order to create a service, an application must first be created.  This is very simple, you only need to specify the name, the namespace and an optional description.
+
+![](/add-app.gif)
+
 ## Creating a service
 
-Creating a service is the main task of the dashboard, as described in other parts of the documentation, 
-an SLA definition must be created for it. This can be easily done using the various input fields 
-in the dashboard. 
+Creating a service is the main task of the dashboard, as described in other parts of the documentation,
```
Comment (nitindermohan) [inline content/docs/dashboard/features.md:30]: Can you add reference to https://www.oakestra.io/docs/getstarted/get-started-app/ highlighting that it is possible to deploy application from the API browser as well but the dashboard makes everything a "one stop shop"?

File: content/docs/dashboard/features.md [hunk 2]
```diff
@@ -16,32 +16,68 @@ The user interface is still under development and therefore does not offer so ma
...
+        "storage": 0,
+        "code": "docker.io/library/nginx:latest",
+        "state": "",
+        "port": "8080",
+        "added_files": []
+        ....
+    }
+  ]
+}
+
```
Comment (nitindermohan) [inline content/docs/dashboard/features.md:69]: The deployment descriptor is different from the example descriptor on the wiki already (https://www.oakestra.io/docs/getstarted/get-started-app/). Can we keep things consistent?

File: content/docs/dashboard/features.md [hunk 3]
```diff
@@ -69,6 +105,31 @@ A user with this role can add resources that can then be used to deploy applicat
 
 This is the default role of a user, he can create applications and services and manage them accordingly.
```
Comment (nitindermohan) [inline content/docs/dashboard/features.md:106]: A description of other roles in dashboard is also needed

File: content/docs/dashboard/features.md [hunk 4]
```diff
@@ -69,6 +105,31 @@ A user with this role can add resources that can then be used to deploy applicat
...
+
+The admin can easily add a new user in the user management and assign him appropriate roles. 
+
+![](/create-user.gif)
+
+## Organizations
+
+Organizations ensure smooth collaboration within a team in Oakestra. 
+Members of an organization have access to all applications created within that organization, 
+and they can use the resources provided within the organization to deploy new services.
```
Comment (nitindermohan) [inline content/docs/dashboard/features.md:118]: Please add a description of "Organization"

File: content/docs/dashboard/features.md [hunk 5]
```diff
@@ -82,3 +143,8 @@ Currently, mails are sent in the following scenarios:
 - And to reset the password of a user.
 
 If nothing is configured, no mails are sent and the admin must reset the password for the user. 
+
+![](/smtp.gif)
+
+
+**Important:** This feater is implemented in the frontend but not yet 100% in the backend and therefore might not work yet.
```
Comment (nitindermohan) [inline content/docs/dashboard/features.md:150]: Is there an issue for it? If no, can we create one and add a link to the issue? That ways once we overhaul we can close the issue and remember to update the wiki as well.

**Review by nitindermohan — CHANGES_REQUESTED** [review]

File: content/docs/dashboard/features.md (hunk 5 above)
Comment (nitindermohan) [reply content/docs/dashboard/features.md:150]: Found the open issue: https://github.com/oakestra/oakestra/issues/182

**Review by nitindermohan — CHANGES_REQUESTED** [review]

**Review by nitindermohan — APPROVED** [review]
Looks good @danimair9

**Pull request merged**